src/Manipulators/Encode.php

```diff
@@ -46,6 +46,7 @@ public function getFormat(Image $image)
             'jpg' => 'image/jpeg',
             'pjpg' => 'image/jpeg',
             'png' => 'image/png',
+            'webp' => 'image/webp',
```

Comment:
@reinink we could check for the existence of the [`imagewebp`](http://www.php.net/manual/en/function.imagewebp.php) function before adding this format.

Reply:
Yes! I like that. Can you add that?

Reply:
I've added the condition, but not sure if it'll work.

Reply:
I had to add it elsewhere 👍